File: Easy/Cpp/1EvenOdd.cpp
// Determining Even/Odd Numbers
// Example:
// Input: number = 4
// Output: Even

#include<iostream>
using namespace std;


bool oddEven(int n){
   
    if(n % 2 == 0){
        return true; 
    } else {
        return false; 
    }
}

int main(){
    int n;
    cout << "Enter a Number: ";
    cin >> n;

    if(oddEven(n)){
        cout << "Even";
    } else {
        cout << "Odd";
    }

    return 0;
}

File: Easy/Py/1EvenOdd.py
#  Determining Even/Odd Numbers
#  Example:
#  Input: number = 4
#  Output: Even

def EvenOdd(number):
    if n%2==0:
        return True
    else:
        return False
    
number=int(input("Enter a number"))
if(EvenOdd(number)):
    print("Even")
else:
    print("Odd")
                